### PR TITLE
Feature/ KIDS 1538 Redesign the Wallet screen

### PR DESCRIPTION
**Pull request by Daniela510**
## Description

- [KIDS-1585](https://nfcollect.atlassian.net/browse/KIDS-1585): Redesign Wallet screen
- [KIDS-1587](https://nfcollect.atlassian.net/browse/KIDS-1587): Implement 'I want to give' bottomsheet.

[KIDS-1585]: https://nfcollect.atlassian.net/browse/KIDS-1585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[KIDS-1587]: https://nfcollect.atlassian.net/browse/KIDS-1587?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/features/profiles/widgets/give_bottomsheet.dart [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app_kids/core/app/pages.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/helpers/analytics_helper.dart';
+import 'package:givt_app_kids/helpers/app_theme.dart';
+import 'package:go_router/go_router.dart';
+
+class GiveBottomsheet extends StatelessWidget {
```
Comment (pavelpavelko) [inline lib/features/profiles/widgets/give_bottomsheet.dart:10]: GiveBottomsheet -> GiveBottomSheet

File: lib/features/profiles/widgets/give_bottomsheet.dart [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+              style: AppTheme.actionButtonStyle
+                  .copyWith(color: AppTheme.givt4KidsBlue),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget givingOptionButton(
```
Comment (pavelpavelko) [inline lib/features/profiles/widgets/give_bottomsheet.dart:79]: '_' at the beginning could help to properly read the code I think

**Review by pavelpavelko — APPROVED** [review]
looks good with a few naming related comments;
Also, the bottom sheet still visible when user goes back from the success screen.

File: lib/features/profiles/widgets/give_bottomsheet.dart (hunk 2 above)
Comment (Daniela510) [reply lib/features/profiles/widgets/give_bottomsheet.dart]: True, but even better if i separate it into a widget. I find this in page approach faster when developing but in our repo we usually separate widgets like this.

**Pull request merged**